vuepress/docs/v7.1/docs/reference/upgrade.md

```diff
@@ -0,0 +1,74 @@
+# Upgrade Guide E7.1.2 ⇒ E7.1.3
+
+## OLM
+
+Follow the steps below to upgrade your Entando OpenShift installation from 7.1.2 to 7.1.3.
+
+1. Open the console Web app of your OpenShift cluster
+
+2. In your EntandoApp namespace, click on the sidebar `Installed Operators`
+
+3. Click on `Entando Operator`
+
+4. Click on `YAML`. A text editor containing YAML arrays should open. Modify the YAML as described below:
+   
+   - For each element of the array `.spec.relatedImages`, update the `sha256` of the `image` property according to [the image-set specification](https://github.com/entando-k8s/entando-k8s-operator-bundle/blob/v7.1.3/values.yaml)
+
+   - In the array `.spec.install.spec.deployments`:
+
+     a. Find the first item with `name` set to "entando-operator"
+
+     b. Go to the `.spec.template.spec.containers[0].env` array
+
+     c. For each `name` element starting with “RELATED_IMAGE_," update the `sha256` of all `value` properties according to [the image-set specification](https://github.com/entando-k8s/entando-k8s-operator-bundle/blob/develop/values.yaml)
+
+     >Note: The `name` property has no **exact** match in the image table, but the matching is still straightforward because every `name` terminates with the uppercase version of the image table key, e.g. `rhel8_mysql_80` maps to `RELATED_IMAGE_RHEL8_MYSQL_80`
```

Comment:
can take out 'the matching is still straightforward because' >> more to the point